tests/unit/pipeline-selector.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { createLocalVue, shallowMount } from '@vue/test-utils';
```

Comment:
General comment: I would prefer folding this commit into the previous one, I would find that more consistent.

Reply:
Done in 9cafa39; I also squashed with the one where I experimented the pipeline selector inside the Vqb component